tests/unit/records/test_api_views.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Zenodo.
+# Copyright (C) 2019 CERN.
+#
+# Zenodo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Unit tests Zenodo JSON deserializer."""
+
+import pytest
+from flask import url_for
+
+from invenio_indexer.api import RecordIndexer
+
+
+@pytest.mark.parametrize(('val', 'status', 'error_message'), [
+    ("2.45,1.63,-1.43,-1.53", 200, None),
+    ("2.45,1.63", 400,
+     "Invalid bounds: Require 4 numbers separated by comma."),
+    ("2.45,'1.63',-1.43,-1.53", 400, "Invalid number in bounds."),
+    ("2.45,' ',-1.43,-1.53", 400, "Invalid number in bounds."),
+    ("2.45,'',-1.43,-1.53", 400, "Invalid number in bounds."),
```

Comment:
I would also add a `2.45,,-1.43,-1.53` (and the single-space equivalent `2.45, ,-1.43,-1.53`). Also, spaces around the numbers would be interesting (i.e. maybe we should `.strip()` them): 
```
1.53  ,    -1.43  ,   2.34,1.23
```

Check also other valid values for the `Decimal` type from the docs, like `NaN`: https://docs.python.org/2/library/decimal.html#quick-start-tutorial